src/extension.ts

```diff
@@ -264,3 +299,24 @@ function getCurrentEnvironmentOrUndefined(): AppsSdkConfigurationEnvironment | u
 export function testsOnly_getImljsonLanguageClient() {
 	return client;
 }
+
+export function sendTelemetry(eventName: string, stringObj?: any, numericObj?: any) {
+	if (isTelemetryEnabled()) {
+		telemetryReporter.sendTelemetryEvent(eventName, stringObj, numericObj);
+	}
+}
+
+export function sendTelemetryError(eventName: string, stringObj?: any, numericObj?: any) {
+	if (isTelemetryEnabled()) {
+		telemetryReporter.sendTelemetryErrorEvent(eventName, stringObj, numericObj);
+	}
+}
+
+export function isTelemetryEnabled() {
+	const _configuration = getConfiguration();
+	if (_configuration.get('telemetry') && _configuration.get('telemetry.disableTelemetry')) {
+		return false;
+	} else {
+		return true;
+	}
+}
```

Comment:
Move to own module? Something like `telemetry.ts`? There can be also the telemetry initialization from top of this file. It will looks like more clear, probably.